adatest/generators.py

```diff
@@ -221,7 +221,7 @@ def __init__(self, models, api_key=None, sep="\n", subsep=" ", quote="\"", tempe
         if api_key is not None:
             openai.api_key = api_key
 
-    def __call__(self, prompts, topic, mode, scorer, num_samples=1, max_length=100):
+    def __call__(self, prompts, topic, mode, scorer=None, num_samples=1, max_length=100):
```

Comment:
I'm concerned that the signatures of the overridden functions don't seem to quite match. I've been trying to make things consistent, but I've not quite managed it.